sorc/chgres_cube.fd/atmosphere.F90

```diff
@@ -1465,8 +1625,19 @@ SUBROUTINE VINTG_WAM (YEAR,MONTH,DAY,HOUR)
 
 ! wind
  print*,"VINTG_WAM:- CALL FieldGet FOR 3-D ADJUSTED WIND."
- call ESMF_FieldGet(wind_target_grid, &
-                    farrayPtr=WIND2PTR, rc=rc)
+
```

Comment:
Should the print statement be updated here too?

Reply:
Done.